plugin.xml

```diff
@@ -39,12 +39,13 @@
       </service>
     </config-file>
     <preference name="ANDROID_SUPPORT_V13_VERSION" default="27.+"/>
-    <preference name="FCM_VERSION" default="17.0.+"/>
+    <preference name="FCM_VERSION" default="17.3.2"/>
```

Comment:
I think google added breaking changes on 17.1.x, so updating to 17.3.2 will probably break the plugin unless we do some code modifications.